DatadogSessionReplay/Sources/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/NodeRecorders/UnsupportedViewRecorder.swift

```diff
@@ -29,7 +31,7 @@ internal struct UnsupportedViewRecorder: NodeRecorder {
         let builder = UnsupportedViewWireframesBuilder(
             wireframeRect: view.frame,
             wireframeID: context.ids.nodeID(for: view),
-            unsupportedClassName: String(reflecting: type(of: view)),
+            unsupportedClassName: context.viewControllerContext.name ?? String(reflecting: type(of: view)),
```

Comment:
suggestion/ The `String(reflecting:)` produces this ugly description for SwiftUI view and I can imagine this being even more odd for more complex SwiftUI contents:
```
SwiftUI._UlHostingView<SHost.(unknown context at $100c1523c).TestView>
```

Why not describing it just as `"SwiftUI"`? The reflected description of its content doesn't bring any value.